docs/develop/make_it_your_own.md

```diff
@@ -280,6 +287,13 @@ curl -k -XPOST -H "Content-Type: application/json" https://localhost:5000/api/re
         "type": "Other",
         "lang": "eng"
     }],
+    "descriptions": [
+        {
+            "description": "A story on how Julio Cesar relates to Gladiator.",
+            "type": "Abstract",
+            "lang": "eng"
+        }
+    ],
```

Comment:
Added issue underlying this: https://github.com/inveniosoftware/invenio-app-rdm/issues/99